services/core-web/src/components/Forms/permits/conditions/AddCondition.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import CustomPropTypes from "@/customPropTypes";
+import AddButton from "@/components/common/AddButton";
+import Section from "@/components/Forms/permits/conditions/Section"
+import SubCondition from "@/components/Forms/permits/conditions/SubCondition"
+import ListItem from "@/components/Forms/permits/conditions/ListItem"
+import { getEditingConditionFlag, getDraftPermitAmendmentForNOW } from "@common/selectors/permitSelectors";
+import { setEditingConditionFlag, createPermitCondition, fetchPermitConditions } from "@common/actionCreators/permitActionCreator";
+
+const propTypes = {
+    handleSubmit: PropTypes.func.isRequired,
+    initialValues: PropTypes.objectOf(PropTypes.any).isRequired,
+    editingConditionFlag: PropTypes.bool.isRequired,
+    setEditingConditionFlag: PropTypes.func.isRequired,
+    createPermitCondition: PropTypes.func.isRequired,
+    fetchPermitConditions: PropTypes.func.isRequired,
+    draftPermitAmendment: CustomPropTypes.permit.isRequired,
+};
+
+const defaultProps = {
+};
+
```

Comment:
defaultProps can be removed if all are required

Reply:
removed